File: laboratorna 3/src/Capsules.java
public class Capsules extends Goods {

    private  double coundInPackaging;

    public Capsules(String name, double price,  int expirationDate, CategoryType categoryType,double coundInPackaging) {
        super(name, price, expirationDate, categoryType );
        this.coundInPackaging = coundInPackaging;
    }

    public double getCoundInPackaging() {
        return coundInPackaging;
    }

    public void setCoundInPackaging(double coundInPackaging) {
        this.coundInPackaging = coundInPackaging;
    }

    @Override
    public String toString() {
        return "Capsules{" + "name='" + super.getName() + '\'' +
                ", price=" + super.getPrice() +
                ", expirationDate=" + getExpirationDate() +
                ", categoryType=" + getCategoryType() +
                "coundInPackaging=" + coundInPackaging +
                '}';
    }
}

File: lab-11 python/Capsules.py
from Goods import Goods

class Capsules(Goods):

    def __init__(self, name, price,  expiration_date, category_type, cound_in_packaging):
        super(Capsules, self).__init__(name, price, expiration_date, category_type)
        self.cound_in_packaging = cound_in_packaging

        def __str__(self):
            return super(Capsules, self).__str__() + ' cound_in_packaging = ' + str(self.cound_in_packaging)